crates/provider/src/ext/trace.rs

```diff
@@ -119,6 +126,15 @@ where
         self.client().request("trace_transaction", (hash,)).await
     }
 
+    async fn trace_get(
+        &self,
+        hash: TxHash,
+        index: u64,
+    ) -> TransportResult<LocalizedTransactionTrace> {
+        // We are using vec![indices] because API accepts a list, but in fact works only if list.len == 1
+        self.client().request("trace_get", (hash, vec![index])).await
```

Comment:
you can use tuples for lists in JSON since they're equivalent to avoid allocating on the heap, see above in `trace_transaction`